cms/static/js/views/settings/grading.js

```diff
@@ -184,9 +206,9 @@ var GradingView = ValidatingView.extend({
             descriptor : this.failLabel(),
             width : nextWidth,
             removable : false
-        }));
+        })) ;
         failBar.find("span[contenteditable=true]").attr("contenteditable", false);
-        gradelist.append(failBar);
+        HtmlUtils.append(gradelist, HtmlUtils.HTML(failBar));
```

Comment:
I reviewed this with @andy-armstrong, and here are the recommendations:
1. Add `contenteditable` as a variable sent to the template gradeCutoffTemplate.  In that way, you don't need `failBar`.
2. Call `HtmlUtils.append(gradelist, this.gradeCutoffTemplate(...))`.
3. Refactor the actual Underscore template string to a separate .underscore file.